contracts/0.4.24/Lido.sol

```diff
@@ -444,10 +444,10 @@ contract Lido is Versioned, StETHPermit, AragonApp {
     }
 
     /**
-    * @notice Send funds to the pool
-    * @dev Users are able to submit their funds by transacting to the fallback function.
+    * @notice Send funds to the pool and mints `StETH` tokens to `msg.sender` address
```

Comment:
Maybe 'send' doesn't collocate with 'mints' here :thinking: 
also, tokens to **the** `msg.sender` address